Adyen/UI/Form/Items/Value Pickers/Identifier Picker/FormAccountIdentifierPickerItem.swift

```diff
@@ -0,0 +1,48 @@
+//
+// Copyright (c) 2025 Adyen N.V.
+//
+// This file is open source and available under the MIT license. See the LICENSE file for more info.
+//
+
+@_spi(AdyenInternal)
+public protocol AccountIdentifier: CustomStringConvertible, Equatable {
```

Comment:
This protocol seems redundant. Also there is an enum below with the same name

Reply:
Removed it